setup.py

```diff
@@ -17,11 +19,18 @@
     url="https://github.com/igmhub/picca",
     author="Nicolas Busca, Helion du Mas des Bourboux et al",
     author_email="nbusca@lpnhe.in2p3.fr",
```

Comment:
I guess for now I'll add my name before the et al. and maybe we can revisit this on a picca task force telecon. I can also change the email since Nicola's LPNHE email is not working anymore. I'll push a commit with these changes

Reply:
I'll also change the email to yours on the github repo settings...